lib/agent.js

```diff
@@ -67,6 +67,18 @@ Agent.prototype.destroy = function () {
   if (this._transport) this._transport.destroy()
 }
 
+Agent.prototype.addPatch = function () {
```

Comment:
If you agree with #918, we should update these functions in a similar fashion

Reply:
Will do. 👍

Reply:
fixed.